tests/process_manager/views/test_partial_views.py

```diff
@@ -193,3 +196,32 @@ def test_filter_table(search, table, expected):
     from process_manager.views.partials import filter_table
 
     assert filter_table(search, table) == expected
+
+
+class HandleErrorsTest(TestCase):
+    """Tests for the HandleErrors decorator in the process_manager views."""
+
+    def setUp(self):
+        """SetUp method to create a RequestFactory instance."""
+        self.factory = RequestFactory()
+
+    @mock.patch("logging.getLogger")
+    def test_exception_view(self, mock_get_logger):
+        """Test the exception_view function."""
+        mock_logger = mock_get_logger.return_value
+
+        @handle_errors
+        def exception_view(request):
+            raise Exception("Test exception")
+
+        request = self.factory.get("/")
+        response = exception_view(request)
+
+        expected_content = render_to_string(
+            "process_manager/partials/error_message.html", request=request
+        )
+        self.assertEqual(response.content.decode(), expected_content)
```

Comment:
All of this should be achievable with using `assertTemplateUsed` and the `client` fixture.